src/sourmash/tax/tax_utils.py

```diff
@@ -340,7 +398,11 @@ def write_classifications(classifications, csv_fp, *, sep=','):
         for res in rank_results:
             rD = res._asdict()
             rD['fraction'] = f'{res.fraction:.3f}'
+            rD['f_weighted_at_rank'] = f'{res.f_weighted_at_rank:.3f}'
```

Comment:
Is there a reason to limit the accuracy of the float to .3f here? Generally I just let numbers be numbers. Not really sure what the consequences of this are, if any... but I wanted to flag it :)

Reply:
I think the ~standard practice is to limit the decimals when printing to stdout, but not while writing to csv? I think that was my original intention --shall I modify to that?

Reply:
oh, yes, I agree. This just looked like csv output to me. Never mind me :)

Reply:
it was ... fixed it to not limit csv. thanks!